action.yml

```diff
@@ -35,28 +33,20 @@ runs:
       with:
         version: v1.68.1
 
-    - name: Decode credentials
-      shell: bash
-      run: |
-        echo $RCLONE_GCS_SERVICE_ACCOUNT_CREDENTIALS_ENCODED | base64 --decode > $HOME/credentials.json
-      env:
-        RCLONE_GCS_SERVICE_ACCOUNT_CREDENTIALS_ENCODED: ${{ inputs.credentials }}
+    - name: AWS OIDC Auth
+      uses: aws-actions/configure-aws-credentials@v4
+      with:
+        aws-region: us-east-2 # Docs are hosted in us-east-2
```

Comment:
Nit: Think it would be cleaner to have this as an input that defaults to `us-east-2` (it's used in two spots here).

Reply:
I could make it a variable. I don't want it as an input because I think changing it would just break things or have unintended behavior. I changed the bucket names to be based on an input flag just to make the action a little easier for us (and because I previously made a typo of the bucket variable name in an action)

Reply:
Turns out to make it a variable for a composite action you have to add a step to export it as an env variable and then access it that way. Not really worth it IMO and I don't want the region to be changeable (even accidentally)